contrib/junit.tpl

```diff
@@ -33,5 +33,16 @@
         </testcase>
     {{- end }}
     </testsuite>
+
+{{- if .Licenses }}
+    {{- $licenses := len .Licenses }}
+    <testsuite tests="{{ $licenses }}" failures="{{ $licenses }}" name="{{ .Target }}" time="0">{{ range .Licenses }}
+        <testcase classname="{{ .PkgName }}" name="[{{ .Severity }}] {{ .Name }}">
+            <failure/>
```

Comment:
Hi @psibre!
I've read again your comment https://github.com/aquasecurity/trivy/discussions/7408#discussioncomment-10488995 and took a look at the JUnit docs...

my concern is `failure` means `failed test`, right?
`failed tests` for `LOW` or `UNKNOWM` license might confuse.
but customers can filter them throught flag `--severity` or rewrite the template.

so, I think we can can approve this PR without additional logic.  

Please, feel free to correct me, because I have no a lot of use cases for JUnit templage.
Thanks!

Reply:
Since Trivy's default behavior is to return exit code 0 even if "failed tests" (in this context: any licenses at all) are found, it should not make a difference whether the license "testcases" are marked as failures or not.
On the other hand, if we _don't_ mark them as failures, then there is no easy way AFAICT of forcing the license scan to actually cause a CI/CD pipeline failure, regardless of severity and custom exit code.